joomla/joomla-cms
# [4.0] ES5 scripts with included polyfills

Pull Request for Issue #25639 .

### Summary of Changes

When I refactored the build tools I left the ES5 babel config basically unconfigured. The reason was I was expecting that we could come up with an action plan (eg a broader talk about polyfills, https://github.com/joomla/joomla-cms/pull/32315#issuecomment-787427272). This obviously didn't happen and so here's the missing configuration.
 

### Testing Instructions
Apply the branch
Run `npm i`
CHECK WITH IE and report any console errors.

**Attention** there are known missing polyfills that this PR **IS NOT** addressing. This PR just ensures modern code is transpilled so IE can run it (this includes also a gazillion of polyfills but there are many DOM related parts that are not polyfillable from babel)

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required
